src/backend/distributed/executor/multi_utility.c

```diff
@@ -630,10 +630,22 @@ ProcessIndexStmt(IndexStmt *createIndexStatement, const char *createIndexCommand
 
 		if (isDistributedRelation)
 		{
+			Oid namespaceId = InvalidOid;
+			Oid indexRelationId = InvalidOid;
 			ErrorIfUnsupportedIndexStmt(createIndexStatement);
 
-			/* if it is supported, go ahead and execute the command */
-			ExecuteDistributedDDLCommand(relationId, createIndexCommand, isTopLevel);
+			/*
+			 * If the command is supported, and the index does not exist,
```

Comment:
Deleting this comment in favor of my other suggestion, which I like better for readability.